clang/bindings/python/clang/cindex.py

```diff
@@ -2888,8 +2896,7 @@ def string(self):
 
         if res:
             return CompletionString(res)
-        else:
-            None
+        return None
```

Comment:
Can you invert this if statement, so that we do early exit on unhappy code paths?